scripts/make_u96v2_sbc_dualcam.sh

```diff
@@ -71,6 +72,10 @@ DEBUG="no"
 #NO_BIT_OPTION can be set to 'yes' to generate a BOOT.BIN without bitstream
 NO_BIT_OPTION='yes'
 
+META_ON_SEMI_URL="https://github.com/Avnet/meta-on-semiconductor.git"
+#META_ON_SEMI_BRANCH="master"
+META_ON_SEMI_BRANCH="u96v2_sbc_dualcam_without_osd"
```

Comment:
Why has "PETALINUX_BUILD_IMAGE" been changed to u96v2-sbc-dualcam-image instead of avnet-image-full?  This breaks consistency with every other BSP.

Reply:
Our image is based on avnet-image-full but we need to add new packages that aren't in it. For example the ap1302 kernel module, the ap1302 firmwares, some scripts... Since the meta-on-semiconductor is a new meta layer and isn't part of the meta it's the cleanest way to do things.

Reply:
I think we can still keep the image name ‘avnet-image-full’ by using IMAGE_INSTALL_append_<> in the <>/meta-avnet/recipes-core/images/avnet-image-full.inc file, similar to what we do for the uz7ev-evcc-hdmi and other designs:

IMAGE_INSTALL_append_uz7ev-evcc-hdmi = "\
    hdmi-utils \
    kernel-module-hdmi \
"

Perhaps something like:
IMAGE_INSTALL_append_u96v2-sbc-dualcam = "\
    <ap1302 kernel module> \
    <ap1302 firmware> \
    <other packages> \
"

My point is that it is already confusing enough for users of our BSPs that they can’t just use the ‘petalinux-build’ command to build their OS image and that they have to modify the command to ‘petalinux-build -c avnet-image-<minimal | full>’.  We have this documented now and even provide rebuild scripts in the BSPs to help users with this.  To change the PETALINUX_BUILD_IMAGE for this BSP makes this BSP a one-off and breaks the consistency we have worked hard to achieve for all the other BSPs.

Reply:
agreed with using "master".  this should actually be 2020.2 eventually.

Reply:
To do what you're saying we need a machine called `u96v2-sbc-dualcam` which we don't have at the time. The difference between the other BSPs and this one is that our camera mezzanine support isn't part of the meta-avnet. To do what you're describing we need to merge the meta-on-semi in the meta-avnet.
At first, the meta-on-semiconductor was also an example to show the customer how to interface with the meta-avnet thus it was designed that way.
What we can do if it's only a naming issue is call our image file `avnet-image-full.bbappend` to override the on in meta-avnet. This way the customer would be able to run `petalinux-build -c avnet-image-full`.

Reply:
"What we can do if it's only a naming issue is call our image file avnet-image-full.bbappend to override the on in meta-avnet. This way the customer would be able to run petalinux-build -c avnet-image-full" -- This seems reasonable.  I like this.

Reply:
Excellent! @AlbertaBeef do you want to do it or should I? We also need to change `PETALINUX_BUILD_IMAGE` in `scripts/make_u96v2_sbc_dualcam.sh` after we make this change

Reply:
@frebaudo-witekio , please do it :)